app/controllers/org_admin/templates_controller.rb

```diff
@@ -387,8 +387,11 @@ def template_options()
               end
             end
           end
-        else
-          # If no funder was selected retrieve the Org's templates
+        end
+        
+        # If the no funder was specified OR the funder matches the org
+        if funder_id.blank? || funder_id == org_id
```

Comment:
we still can end up passing a blank org_id to the scope method if for any reason the params is not present...